src/data/battler-tags.ts

```diff
@@ -421,16 +428,13 @@ export class BeakBlastChargingTag extends BattlerTag {
    * to be removed after the source makes a move (or the turn ends, whichever comes first)
    * @param pokemon {@linkcode Pokemon} the owner of this tag
    * @param lapseType {@linkcode BattlerTagLapseType} the type of functionality invoked in battle
-   * @returns `true` if invoked with the CUSTOM lapse type; `false` otherwise
+   * @returns `true` if invoked with the ON_GET_HIT lapse type; `false` otherwise
```

Comment:
```suggestion
   * @returns `true` if invoked with the `AFTER_HIT` lapse type; `false` otherwise
```
Oops, missed this before.